packages/server/locales/es.json

```diff
@@ -1,275 +1,1272 @@
+	"Columns": "Columnas",
+	"Default state": "Default state",
+	"Specify the fields in the table to show": "Specify the fields in the table to show",
+	"Use view to create": "Use view to create",
+	"If user has write permission. Leave blank to have no link to create a new item": "If user has write permission. Leave blank to have no link to create a new item",
+	"Display create view as": "Display create view as",
+	"Label for create": "Label for create",
+	"Label in link or button to create. Leave blank for a default label": "Label in link or button to create. Leave blank for a default label",
+	"Fixed fields": "Fixed fields",
+	"These fields were missing, you can give values here. The values you enter here can be overwritten by information coming from other views, for instance if the form is triggered from a list.": "These fields were missing, you can give values here. The values you enter here can be overwritten by information coming from other views, for instance if the form is triggered from a list.",
+	"Edit options": "Edit options",
+	"The view you choose here can be ignored depending on the context of the form, for instance if it appears in a pop-up the redirect will not take place.": "The view you choose here can be ignored depending on the context of the form, for instance if it appears in a pop-up the redirect will not take place.",
+	"View when done": "View when done",
+	"Item View": "Item View",
+	"If user has write permission.  Leave blank to have no link to create a new item": "If user has write permission.  Leave blank to have no link to create a new item",
+	"Subtables": "Subtables",
+	"List View": "List View",
+	"Show View": "Show View",
+	"Which related tables would you like to show in sub-lists below the selected item?": "Which related tables would you like to show in sub-lists below the selected item?",
+	"Order and layout": "Order and layout",
+	"Order by": "Order by",
+	"Descending": "Descending",
+	"Columns small screen": "Columns small screen",
+	"Columns medium screen": "Columns medium screen",
+	"Columns large screen": "Columns large screen",
+	"Columns extra-large screen": "Columns extra-large screen",
+	"Each in card?": "Each in card?",
+	"Omit search form": "Omit search form",
+	"Do not display the search filter form": "No mostrar el formulario de filtro de búsqueda",
+	"Default search form values when first loaded": "Valores del formulario de búsqueda predeterminado cuando se carga por primera vez",
+	"Next":"Siguiente",
+	"Save": "Guardar",
+	"Calculated fields cannot have File type": "Los campos calculados no pueden tener tipo de archivo,"
```

Comment:
comma (,) should be outside quotes, otherwise this is invalid JSON. There are more of these, i see the same in lines 363-367. You can check the whole file by pasting here: https://jsonlint.com/

Reply:
Done